tuna/machine.py

```diff
@@ -151,7 +151,7 @@ def connect(self, abort=None):
         'user': self.user,
         'password': self.password,
         'port': self.port,
-        'local_machine': self.local_machine,
+        'local_machine': self.local_machine,  # pylint: disable=no-member ; false alarm
```

Comment:
i believe this can be avoided if in the class declaration we set self.local_machine = None. This way pylint will see it as a member. Please try this out and let me know. We should not be having disable-no-member anywhere. This is an easy fix

Reply:
would it be good to differentiate b/w a member that is defined but set to `None` vs. a member not defined at all?

Reply:
all members need to be defined. At least set to None.